app/views/layouts/_flash.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if flash[:alert] || flash[:notice] %>
+  <div class="flash-<%= flash.first.first %>">
```

Comment:
I don't know what this returns. What does this do?

Reply:
I guess the key of the first object in the flash hash?